.tutone.yml

```diff
@@ -592,6 +592,8 @@ packages:
         create_as: string
       - name: Seconds
         create_as: string
```

Comment:
Do we know that `Seconds` as string won't bite us either?

Reply:
We do not.  Minutes is a scalar, but comes back in the JSON as an int.

Reply:
Seconds also is a scalar.  I'll look for some usage of it.

Reply:
https://github.com/newrelic/newrelic-client-go/issues/617